packages/blade/src/components/Input/TextArea/__tests__/TextArea.native.test.tsx

```diff
@@ -0,0 +1,130 @@
+import { fireEvent } from '@testing-library/react-native';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import { TextArea } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
```

Comment:
parity this with `TextInput.native.test`

Reply:
Done.